provider/terraform/resources/resource_compute_backend_service.go.erb

```diff
@@ -535,7 +539,9 @@ func expandBackendService(d *schema.ResourceData) (*computeBeta.BackendService,
 				ForceSendFields: []string{"IncludeProtocol", "IncludeHost", "IncludeQueryString", "QueryStringWhitelist", "QueryStringBlacklist"},
 			},
 		},
+		<% if version.nil? || version == 'ga' -%>
```

Comment:
This should be unless

Reply:
Oh yup - I'd fixed that locally but managed to not push it somehow. Fixed.